src/langchain_google_spanner/vector_store.py

```diff
@@ -276,7 +276,7 @@ def init_vector_store_table(
         instance_id: str,
         database_id: str,
         table_name: str,
-        client: spanner.Client = spanner.Client(),
+        client: Optional[spanner.Client] = None,
```

Comment:
Now that this is None we need to get a client via `client_with_user_agent(client, USER_AGENT_VECTOR_STORE)`